kinvolk/habitat-service-broker
# Run multiple instances

**Prerequisite**

https://github.com/habitat-sh/habitat-operator/pull/307

**Test instructions**

After deploying the broker, running `kubectl apply -f manifests/redis/service-instance.yaml` should start three pods in the namespace `redis-habitat`.

Apply the binding and test for replication.

Related to #22 

**Note**
This branch should not be deleted post merge, as https://github.com/habitat-sh/habitat/issues/5264 references it for reproducing the issue.
